indico/modules/rb/models/reservations.py

```diff
@@ -274,6 +278,10 @@ def is_archived(self):
     def is_repeating(self):
         return self.repeat_frequency != RepeatFrequency.NEVER
 
+    @hybrid_property
```

Comment:
This should be a normal property; it wouldn't work as-is in hybrid mode anyway (`is` and `and` cannot be overridden by a class even with dunders or other magic so you'd need an explicit .expression method for it).

And FWIW, the fact that you added this method shows exactly why we generally don't use nullable string columns. ;)
There's a good chance you don't need it anymore in the end, but if you do, the whole body can be simplified to `return bool(self.notes)`

Reply:
Ah, I see. I wrote this property for future use in the backend, but after seeing your previous comment on the migration, I'll probably just make it return the bool for now (otherwise, I'll just remove it entirely).